kubescaler/scaler/aws/cluster.py

```diff
@@ -78,15 +82,16 @@ def __init__(
         self.kube_config_file = kube_config_file or "kubeconfig-aws.yaml"
         self.image_ami = get_latest_ami(self.region, self.kubernetes_version)
         self.machine_type = self.machine_type or "m5.large"
+        self.ami_type = ami_type or "AL2_x86_64"
```

Comment:
I'm learning in the Go SDK there are API ways to derive this (e.g., the other likely option is `AL2_ARM_64` but I'm good avoiding using extra AWS APIs for now. :laughing:

Reply:
in future, we may use the launch template. in that case, we will not use ami type altogether. so, I think, we can leave it as it is now and then when we decides to use launch template, we can redesign it?

Reply:
oh yeah I'm not saying we should change this now - just sharing something I learned this morning.